OrbitGl/FramePointerValidatorClient.cpp

```diff
@@ -75,6 +68,6 @@ void FramePointerValidatorClient::HandleResponse(const Message& message,
 
   std::string text =
       absl::StrFormat("Failed to validate %d out of %d functions",
-                      std::get<1>(response).size(), num_functions);
+                      num_fpo_functions, num_functions);
```

Comment:
what if all functions are compliant? consider showing something more positive in this case. Failed to validate 0 out of 10000 functions does not sound as a success.

Maybe change the message to something like
"Validation complete
Module \"%s\": %d functions support frame pointers, %d functions don't.
...."

Reply:
Renamed it, however I kept the summary instead of a dialogue for each module.

Reply:
After discussed offline, I renamed it again, to give the overview per module.